example/example.go

```diff
@@ -54,21 +51,9 @@ func printAMI(fcosstable stream.Stream) error {
 }
 
 func run() error {
-	streamurl := fedoracoreos.GetStreamURL(fedoracoreos.StreamStable)
-	resp, err := http.Get(streamurl.String())
+	fcosstable, err := fedoracoreos.FetchStream(fedoracoreos.StreamStable)
```

Comment:
Minor/optional: `fcosstable` looks weird. Maybe `fcos_stable` instead?

Reply:
Done and pushed another minor commit here.